packages/@aws-cdk/aws-certificatemanager/README.md

```diff
@@ -81,6 +81,64 @@ const cert = new acm.Certificate(this, 'Certificate', {
 });
 ```
 
+If the Route 53 Hosted Zone is in a separate AWS account you can pass a role to assume to create the validation records
+
+```ts
+const example.vom = route53.HostedZone.fromHostedZoneAttributes(this, 'ExampleCom', {
+  hostedZoneId: 'Z00123456789',
+  zoneName: 'example.com',
+});
+
+const roleArn = Stack.of(this).formatArn({
+  region: '',
+  service: 'iam',
+  account: '123456789', // Role of account with zone
+  resource: 'role',
+  resourceName: 'DNSRole', // Role that is able to make changes
+});
+const role = iam.Role.fromRoleArn(this, 'Role', roleArn);
+
+const cert = new acm.DnsValidatedCertificate(this, 'Certificate', {
+  domainName: 'example.com',
+  hostedZone: zone,
+  dnsRole: role,
+});
+```
+
+The role in the account with the hosted zone needs to look like
+
+```ts
+ const role = new iam.Role(this, 'Role', {
```

Comment:
I think there needs to be a factory for this role. This feels a bit like implementation detail leakage.

Reply:
I have a construct I've released which handles the role creation. https://constructs.dev/packages/cdk-cross-account-route53/
I wasn't sure if it was appropriate to include it in CDK itself.